tests/unit/CMakeLists.txt

```diff
@@ -45,6 +45,7 @@ target_include_directories(sentry_test_unit PRIVATE
 target_link_libraries(sentry_test_unit PRIVATE
 	${SENTRY_LINK_LIBRARIES}
 	${SENTRY_INTERFACE_LINK_LIBRARIES}
+	"$<$<PLATFORM_ID:Linux>:rt>"
```

Comment:
can you add a comment what this means? I have no idea TBH

Reply:
Links `rt` library if target platform is Linux

https://cmake.org/cmake/help/latest/manual/cmake-generator-expressions.7.html

https://docs.oracle.com/cd/E88353_01/html/E37842/librt-3lib.html

Reply:
well yes, but why is that suddenly needed for the unit tests only? Is that a restriction of old GCC?

Reply:
Yes, older gccs don't automatically link librt when needed. Im not sure what version started handling it automatically.